### PR TITLE
Vaccines Reorgnization

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This PR moves text around to try to streamline the description of the vaccine candidates. Instead of walking through each platform, it will instead lay out the mechanism of each approach and some efforts to use it for COVID-19. Then, safety will be summarized across all of the vaccine candidates, and efficacy will be described across all platforms as well. 

I think this gives us more flexibility to discuss things like response to variants and boosters, while also contextualizing some of the information a little more effectively (what does it mean have to 23% of people have an adverse reaction to a particular vaccine? Is that high or low?)

There are lots of blanks to fill in, and the initial descriptions need to be finessed, but I wanted to put this up sooner than later since it is the kind of thing that causes merge conflicts!

It's still long (13,364 words) so I'm going to try to move some other content to the Appendix, as well!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/22.vaccines.md [hunk 1]
```diff
@@ -313,14 +283,15 @@ The main advantage of this method is that protein subunit vaccines are considere
...
+There have been examples of success in preclinical research including candidate RBD219N-1, a 218-amino-acid residue of the SARS-CoV-1 RBD that, when adjuvanted to aluminum hydroxide, was capable of eliciting a high RBD-specific and neutralizing antibody response in both pseudovirus and live virus infections of immunized mice [@doi:10.4161/hv.27464].
+Another set of studies has examined the immunogenicity of a SARS-CoV-1 RBD fused with IgG1 Fc.
+This recombinant fusion protein could induce a robust long-lasting neutralizing antibody and cellular immune response that protected mice from SARS-CoV-1 [@doi:10.3389/fimmu.2020.602256; @doi:10.1016/j.vaccine.2006.10.031; @doi:10.1016/j.vaccine.2006.06.084].
+While there have been other potential protein subunit vaccines for SARS-CoV-1 investigated _in vivo_ [@10.3389/fimmu.2020.602256; @doi:10.3389/fmicb.2020.00298], none of these candidates have successfully completed clinical trials, more than likely due to the fact that the SARS-CoV-1 epidemic mostly ended by May 2004, and there was thus less of a demand for SARS-CoV-1 vaccine research.
```
Comment (RLordan) [inline content/22.vaccines.md:294]:
```suggestion
While there have been other potential protein subunit vaccines for SARS-CoV-1 investigated _in vivo_ [@10.3389/fimmu.2020.602256; @doi:10.3389/fmicb.2020.00298], none of these candidates have successfully completed clinical trials, more than likely due to the fact that the SARS-CoV-1 epidemic mostly ended by May 2004, and there was thus less of a demand or funding for SARS-CoV-1 vaccine research. 
```

**Review by RLordan — APPROVED** [review]
I am a fan of the reorganization. Much more intuitive.

**Comment by RLordan** [comment]
Also liking the table added.

**Comment by rando2** [comment]
Thank you so much @RLordan! I'm glad it makes sense. I'm working on getting some of the maps we had talked about set generated and then I'll go back to working on the text! I feel like we're getting close, we can do it!

**Pull request merged**